### PR TITLE
Fix/PN-13071 - Change how SERCQ component is displayed when PEC is enabled (PF and PG)

### PR DESCRIPTION
**Pull request by mflauti**
## Short description
This PR fixes a bug in SERCQ display mode when PEC is enabled as default legal contact for PF and PG

## How to test
Login into PF and PG (as Admin), add a PEC as default legal contact (wait for it to switch in 'verified' status) and check the SERCQ component is displayed as expected following its figma specs.

### CODE REVIEW

File: packages/pn-personafisica-webapp/src/components/Contacts/SercqSendContactItem.tsx [hunk 1]
```diff
@@ -298,17 +303,27 @@ const SercqSendContactItem: React.FC = () => {
...
+      }
+      subtitle={
+        defaultPECAddress?.pecValid
+          ? t('legal-contacts.sercq-send-description-pec-enabled', { ns: 'recapiti' })
+          : t('legal-contacts.sercq-send-description', { ns: 'recapiti' })
       }
-      subtitle={t('legal-contacts.sercq-send-description', { ns: 'recapiti' })}
       expanded
       sx={{
-        borderBottomLeftRadius: value ? 0 : 4,
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/components/Contacts/SercqSendContactItem.tsx:306]: Why did you remove this?

File: packages/pn-personafisica-webapp/src/components/Contacts/SercqSendContactItem.tsx [hunk 2]
```diff
@@ -298,17 +303,27 @@ const SercqSendContactItem: React.FC = () => {
...
-        borderBottomRightRadius: value ? 0 : 4,
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/components/Contacts/SercqSendContactItem.tsx:307]: Why did you remove this?

File: packages/pn-personagiuridica-webapp/src/components/Contacts/PecContactItem.tsx [hunk 3]
```diff
@@ -156,6 +156,8 @@ const PecContactItem: React.FC = () => {
         pt: defaultSERCQ_SENDAddress ? 0 : 3,
         borderTopLeftRadius: defaultSERCQ_SENDAddress ? 0 : 4,
         borderTopRightRadius: defaultSERCQ_SENDAddress ? 0 : 4,
+        borderBottomLeftRadius: defaultPECAddress?.pecValid ? 0 : 4,
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/components/Contacts/PecContactItem.tsx:159]: here you can use value

File: packages/pn-personagiuridica-webapp/src/components/Contacts/PecContactItem.tsx [hunk 4]
```diff
@@ -156,6 +156,8 @@ const PecContactItem: React.FC = () => {
...
+        borderBottomRightRadius: defaultPECAddress?.pecValid ? 0 : 4,
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/components/Contacts/PecContactItem.tsx:160]: here you can use value

File: packages/pn-personagiuridica-webapp/src/components/Contacts/SercqSendContactItem.tsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import { useEffect, useRef, useState } from 'react';
 import { useTranslation } from 'react-i18next';
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/components/Contacts/SercqSendContactItem.tsx:2]: Do the same changes requested for pf

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-personafisica-webapp/src/components/Contacts/SercqSendContactItem.tsx (hunk 2 above)
Comment (mflauti) [reply packages/pn-personafisica-webapp/src/components/Contacts/SercqSendContactItem.tsx:307]: See above

File: packages/pn-personagiuridica-webapp/src/components/Contacts/PecContactItem.tsx (hunk 3 above)
Comment (mflauti) [reply packages/pn-personagiuridica-webapp/src/components/Contacts/PecContactItem.tsx:159]: Do you mean 'currentValue'? But here I'm checking if the pec address has been validated because if it's been already validated it needs to be displayed above sercq component and then it do not need to have border radius on the bottom

File: packages/pn-personafisica-webapp/src/components/Contacts/SercqSendContactItem.tsx (hunk 1 above)
Comment (AndreaCimini90) [reply packages/pn-personafisica-webapp/src/components/Contacts/SercqSendContactItem.tsx:314]: It should be wrong. Because the borderBottom props was used to rendere the way in the case fo sercq enabled

File: packages/pn-personagiuridica-webapp/src/components/Contacts/PecContactItem.tsx (hunk 3 above)
Comment (AndreaCimini90) [reply packages/pn-personagiuridica-webapp/src/components/Contacts/PecContactItem.tsx:159]: value is empty if pec is not valid

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]
The layout SERCQ enabled and PEC in validation and SERCQ not enabled and PEC in validation is different from figma https://www.figma.com/design/DlzvNQzkecy7L2ym40OTge/Domicilio-digitale?node-id=1895-36651&node-type=section&t=hvHTPmE8UbllxmD2-0

**Review by AndreaCimini90 — APPROVED** [review]
